Lib/test/test_minidom.py

```diff
@@ -1663,5 +1664,14 @@ def test_cdata_parsing(self):
         dom2 = parseString(dom1.toprettyxml())
         self.checkWholeText(dom2.getElementsByTagName('node')[0].firstChild, '</data>')
 
+    def testQuoteEscape(self):
+        text = ElementTree.Element('text')
+        text.text = 'f&oo"b<a>r'
+        xml_string = ElementTree.tostring(text)
```

Comment:
We should not rely on an unrelated library providing the same text string serialisation. Instead, the test should use a plain string to compare against. Otherwise, changes in ElementTree could make this test fail without need.